.travis.yml

```diff
@@ -27,11 +19,33 @@ branches:
 matrix:
     fast_finish: true
     include:
+          # Minimum supported PHP and Symfony version
         - php: 5.4
-          env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest" COVERAGE=true TEST_COMMAND="composer test-ci"
-
+          env: DEPENDENCIES="minimum" COVERAGE=true TEST_COMMAND="composer test-ci"
+
+          # Test the latest stable release
+        - php: 7.1
+
+          # Test LTS versions
+        - php: 7.1
```

Comment:
Food for thought: test the LTS releases against the lowest common version they and this package supports. I'd say that people on LTS releases are less likely to always run on the latest PHP version, so testing against a lower version might be more comparable to real-world usage.